### PR TITLE
Confirmation/Response on adding WCW topic

### PR DESCRIPTION
**Pull request by DarrylBrooks97**
Screenshot 2022-11-10 at 11 43 39 AM

### CODE REVIEW

File: src/commands/update-wcw.ts [hunk 1]
```diff
@@ -23,19 +24,65 @@ export async function execute(interaction: CommandInteraction<CacheType>) {
...
-            icon: '🟢',
-            notify: true,
-        }),
-    ]);
+    await interaction.reply({
+        content: `Are you sure you want to add the topic: ${topic}?`,
+        components: ConfirmationButton,
+    });
+
+    const filter = (i: any) => i.customId === 'confirm' || i.customId === 'cancel';
```
Comment (gt-codes) [inline src/commands/update-wcw.ts]: Can we remove the anys here? @DarrylBrooks97

**Review by gt-codes — CHANGES_REQUESTED** [review]
Looks good aside from the anys from the iterator.

File: src/commands/update-wcw.ts (hunk 1 above)
Comment (DarrylBrooks97) [reply src/commands/update-wcw.ts]: Just typed it, removing any just threw errors @gt-codes

**Review by Genyus — APPROVED** [review]
👍🏾

**Pull request merged**